inc/acf/admin/post-display-settings.php

```diff
@@ -9,6 +9,8 @@
 
 add_action('acf/post_type/render_settings_tab/display-settings', function ($acf_post_type) {
 
+    $post_label = $acf_post_type['labels']['singular_name'];
+
```

Comment:
In case for some reason it can't get the label I'd write like this $post_label = $acf_post_type['labels']['singular_name'] ?? null;

Reply:
Done